File: Java/PB with Java/src/PetStore.java
import java.util.Scanner;

public class PetStore {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter count of dog food packages: ");
        int countForDogs = Integer.parseInt(scanner.nextLine());
        System.out.print("Enter count of cat food packages: ");
        int countForCats = Integer.parseInt(scanner.nextLine());
        double priceForDogs = 2.50 * countForDogs;
        double priceForCats = 4 * countForCats;
        double sum = priceForDogs + priceForCats;
        System.out.print("The sum is $" + sum);
    }
}

File: C++/Programming Basics with C++/First Steps In Programming - Lab and Exc/PetStore/PetStore.cpp
#include <iostream>

int main() {
    int countForDogs, countForCats;

    std::cout << "Enter count of dog food packages: ";
    std::cin >> countForDogs;
    std::cout << "Enter count of cat food packages: ";
    std::cin >> countForCats;

    double priceForDogs = 2.50 * countForDogs;
    double priceForCats = 4.0 * countForCats;
    double sum = priceForDogs + priceForCats;

    std::cout << "The sum is $" << sum;

    return 0;
}
